src/events/message_create/praise.ts

```diff
@@ -15,15 +21,22 @@ export default async function handler(message: Message) {
 		return;
 	if (message.member === null) return;
 
-	try {
-		if (message.content.match(ENABLE_PATTERN)) {
-			await message.member!.roles.add(WACKY_ROLE_ID);
-			await message.react("🥳");
-		} else if (message.content.match(DISABLE_PATTERN)) {
-			await message.member!.roles.remove(WACKY_ROLE_ID);
-			await message.react("🤐");
-		}
-	} catch (e) {
-		console.error(e);
-	}
+  try {
+    for (const ENABLE_PATTERN of ENABLE_PATTERNS) {
+      if (message.content.match(ENABLE_PATTERN)) {
+        await message.member!.roles.add(WACKY_ROLE_ID);
+        await message.react("🥳");
+        return;
+      }
+    }
+
+    for (const DISABLE_PATTERN of DISABLE_PATTERNS) {
+      if (message.content.match(DISABLE_PATTERN)) {
+        await message.member!.roles.remove(WACKY_ROLE_ID);
+        await message.react("🤐");
+      }
```

Comment:
May want a `break`/`return` here. If both patterns match, removing the role twice will probably throw an error.

Reply:
Fixed, thanks!